docs/start/quickstart.md

```diff
@@ -19,7 +19,7 @@ cd my-remix-app
 npm i @remix-run/node @remix-run/react @remix-run/serve isbot react react-dom
 
 # install dev dependencies
-npm i -D @remix-run/dev typescript
+npm i -D @remix-run/dev typescript @types/react
```

Comment:
Been a bit since I did the tutorial, but does this need/want `@types/react-dom` as well?

```suggestion
npm i -D @remix-run/dev typescript @types/react @types/react-dom
```

Reply:
Oh yeah, good point! I'll update it

Reply:
Ended up removing these again and `typescript` as well, since the quickstart should not have TypeScript`